pytorch_generative/models/autoregressive/__init__.py

```diff
@@ -0,0 +1 @@
+# empty __init__.py for easier access to models
```

Comment:
All the models are already importable with `pytorch_generative.models.<model-name>`. Is that not working for you?